nemo/collections/nlp/models/language_modeling/megatron_gpt_model.py

```diff
@@ -314,78 +314,86 @@ def configure_gradient_clipping(self, *args, **kwargs):
         parameters = self.model.parameters()
         clip_grad_norm_fp32(parameters=parameters, max_norm=clip_val)
 
+    def buckets(self, batch):
```

Comment:
it looks like this should be a class method not an instance method. Or maybe it should go to https://github.com/NVIDIA/NeMo/tree/main/nemo/collections/nlp/data/language_modeling/megatron/....  @ericharper  what do you think?

Reply:
Also rename it to something like "bucketize_for_gpt_inference"